README.md

```diff
@@ -25,12 +28,15 @@ module "ipa0" {
 module "ipa1" {
   source = "github.com/cisagov/freeipa-server-tf-module"
 
-  domain               = "example.com"
-  hostname             = "ipa.example.com"
-  ip                   = "10.10.10.5"
-  security_group_ids   = ["sg-51530134", "sg-51530245"]
-  subnet_id            = aws_subnet.second_subnet.id
-  tags                 = {
+  domain              = "example.com"
+  hostname            = "ipa.example.com"
```

Comment:
Just a quick check, should both modules in the example have the same `hostname`?

Reply:
Great catch!  See commit c3c050a.